plugin/main/src/kotlinx/benchmark/gradle/Utils.kt

```diff
@@ -244,6 +249,7 @@ private object ValidOptions {
     )
     val modes = setOf("thrpt", "avgt", "Throughput", "AverageTime")
     val nativeForks = setOf("perBenchmark", "perIteration")
+    val jvmProfilers = setOf("stack", "gc", "cl", "comp")
```

Comment:
Should we support other profilers described here? https://github.com/openjdk/jmh/blob/362d6579e007f0241f05c1305f0b269fcc2cc27a/jmh-samples/src/main/java/org/openjdk/jmh/samples/JMHSample_35_Profilers.java#L352C97-L352C97

Reply:
Those profilers seem to require root privileges to run (at least for macOS). Such requirements can be documented in the configuration options document